Makefile

```diff
@@ -0,0 +1,82 @@
+.PHONY: db
+
+DATA_LINK="https://s3.amazonaws.com/instacart-datasets/instacart_online_grocery_shopping_2017_05_01.tar.gz"
+# Overwrite this by running `make BUCKET="my-test-bucket" s3` for example
+BUCKET="instacart-store"
+# MODE can be "local" or "rds"
+# Overwrite this by running `make MODE="rds" db` for example
+MODE="local"
+
+data/external/raw_data.tar.gz:
+	curl -X GET ${DATA_LINK} -o data/external/raw_data.tar.gz && tar -xvzf data/external/raw_data.tar.gz -C data/external && mv data/external/instacart_2017_05_01/* data/external && rm -rf data/external/instacart_2017_05_01
+
+data: data/external/raw_data.tar.gz
+
+data/external/aisles.csv: data
```

Comment:
Is it necessary to have make commands for every single data file?

Reply:
I will separate out the two separate data files into their own directives

Reply:
Actually I misunderstood this comment - the various csv files you see here are a result of unpacking the compressed data.tar.gz which is downloaded from instacart source. They all come together so there is no sense in creating them individually - the only reason they are all listed out here is because feature_gen requires each of those files to be in place, so if any is missing, the makefile will fall back to `make data` directive to re-download raw source